src/box/iproto_constants.h

```diff
@@ -218,7 +218,22 @@ extern const char *iproto_flag_bit_strs[];
 	/**
 	 * Flag indicating whether the transaction is synchronous.
 	 */								\
-	 _(IS_SYNC, 0x61, MP_BOOL)
+	 _(IS_SYNC, 0x61, MP_BOOL)					\
+	 /**
+	  * Flag indicating whether checkpoint join should be done.
+	  */								\
+	 _(CHECKPOINT_JOIN, 0x62, MP_BOOL)				\
```

Comment:
I guess it should be named `IS_CHECKPOINT_JOIN` according to our naming conventions.

Reply:
I tried to be consistent with `FETCH_POSITION`, `REPLICA_ANON`. The only flag prefixed with `IS_` is `IS_SYNC`. Not really sure, that prefixing in iproto flags is out naming convention

Reply:
Done this actually